src/app/app.module.ts

```diff
@@ -0,0 +1,51 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { HttpModule } from '@angular/http';
+import { RouterModule } from '@angular/router';
+
+import { AppComponent } from './app.component';
+import { LoginComponent } from './login/login.component';
+import { NavbarComponent } from './navbar/navbar.component';
+import { SignupComponent } from './signup/signup.component';
+import { HomepageComponent } from './homepage/homepage.component';
+import { FooterComponent } from './footer/footer.component';
+import { ContactComponent } from './contact/contact.component';
+
+@NgModule({
+  declarations: [
+    AppComponent,
+    LoginComponent,
+    NavbarComponent,
+    SignupComponent,
+    HomepageComponent,
+    FooterComponent,
+    ContactComponent
+  ],
+  imports: [
+    BrowserModule,
+    FormsModule,
+    HttpModule,
+    RouterModule.forRoot([
```

Comment:
Nice use of the router!

Reply:
Thanks, the instructional video you linked me really helped!